### ISSUE TITLE
Missing keeper in update variants for `enable_gcfs` node pool option

### ISSUE THREAD

**Issue by lauraseidler** [terraform-google-modules/terraform-google-kubernetes-engine#1217]
### TL;DR

Adding the `gcfs_config` block (controlled through `enable_gcfs` option for node pools) forces the node pool to get re-created, but the keeper is missing for the update variants, causing the apply to fail.

### Expected behavior

New node pool gets created, and then the old node pool gets destroyed.

### Observed behavior

Upgrade fails, as node pool with the same name already exists.

### Terraform Configuration

```hcl
Any update-variant cluster with a node pool that was created before #1174 got merged
```

### Terraform Version

```sh
$ terraform version      
Terraform v1.1.8
on linux_amd64
+ provider registry.terraform.io/hashicorp/external v2.2.2
+ provider registry.terraform.io/hashicorp/google v4.10.0
+ provider registry.terraform.io/hashicorp/google-beta v4.10.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.9.0
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.1.2
+ provider registry.terraform.io/hashicorp/time v0.7.2
```

### Additional information

I'll provide a PR for this shortly.

Note: Enabling/disabling image streaming via gcloud is possible without re-creating the node pool, but this doesn't seem to be supported by the current provider version.
